test/configure.js

```diff
@@ -29,7 +29,13 @@ suite( 'Configure: Settings', () => {
         }, data => {
 
             assert.isArray( data, 'Sugarcoat returns an array when we are missing a dest.' );
-            assert.propertyVal( data[0], 'key', 'settings.dest', 'We have dest in the error array');
+
+            data.forEach( ( errorObj, index ) => {
+
+                assert.instanceOf( errorObj, Error, 'The object was an Error Object.' );
```

Comment:
boom. Moved the errors into a separate file. That way we have them all in one place, and we can use that for testing as well.